src/javascript/binary/websocket_pages/user/account/authenticate.js

```diff
@@ -23,6 +32,236 @@ const Authenticate = (() => {
         });
     };
 
+    const init = () => {
+        // Setup accordion
+        $('.files').accordion({
+            heightStyle: 'content',
+            collapsible: true,
+            active     : false,
+        });
+        // Setup Date picker
+        const tomorrow = new Date();
+        tomorrow.setDate(tomorrow.getDate() + 1);
+        $('.date-picker').datepicker({
+            dateFormat : 'yy-mm-dd',
+            changeMonth: true,
+            changeYear : true,
+            minDate    : tomorrow,
+        });
+
+        // Submit button
+        let $button;
```

Comment:
can we define this somewhere above `showSubmit` function as it's not used anywhere from line 53 to 101?